tests/sparsezoo/analysis/test_helpers.py

```diff
@@ -0,0 +1,368 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
great job on tests, very thorough. let's do an audit of how long the test suite takes to run (sparsezoo downloads can sometimes be a bottleneck) and the range of cases tested in a pair programming session

Reply:
Thanks for the review Benjamin!